devicemodel/hw/pci/virtio/vhost_vsock.c

```diff
@@ -181,23 +181,23 @@ vhost_vsock_init(struct virtio_base *base, int vq_idx)
 	vhost_vsock->vdev.vqs = vhost_vsock->vqs;
 	vhost_vsock->vhost_fd = open("/dev/vhost-vsock", O_RDWR);;
 	if (vhost_vsock->vhost_fd < 0) {
-		pr_err(("Open vhost-vsock fail, pls open vsock kernel config!\n"));
+		pr_err(("Open vhost-vsock fail, pls open vsock kernel config.\n"));
```

Comment:
We should rather ask the user to check his kernel config and point at the configuration variable to check on.